fetch.bs

```diff
@@ -2123,11 +2125,11 @@ clearly stipulates that <a>connections</a> are keyed on
      WebSocket saner -->
 
 
-<h3 id=http-cache-partitions>HTTP cache partitions</h3>
+<h3 id=http-state-partition>HTTP state partition key</h3>
```

Comment:
Should we make this network or perhaps fetch partition key? Network might be better as we'll want to use it for things that do not go through Fetch as well, such as RTCDataChannel, right?